gateway/proxy_eth.go

```diff
@@ -634,6 +634,14 @@ func (gw *Node) EthTraceTransaction(ctx context.Context, txHash string) ([]*etht
 	return gw.target.EthTraceTransaction(ctx, txHash)
 }
 
+func (gw *Node) EthTraceFilter(ctx context.Context, filter ethtypes.EthTraceFilterCriteria) ([]*ethtypes.EthTraceFilterResult, error) {
```

Comment:
you also need to run `filter.FromBlock` and `filter.ToBlock` through `gw.checkBlkParam` to make sure we stick to lookback limits

Reply:
see `EthGetLogs` for example

Reply:
ac53ddbec

Reply:
ac53ddbec